### DISCUSSION TITLE
Question: fnf modding errors

### DISCUSSION THREAD

**Discussion by FelineStudios2021** [FunkinCrew/Funkin#1740] [Compiling help] [answered]
I have a problem :( Plz help me!

C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/ParseRules.hx:287: characters 31-35 : UnicodeString has no field iter
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/ParseRules.hx:386: characters 31-35 : UnicodeString has no field iter
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 7-8 : Unknown identifier : f

**Comment by Cheemsandfriends** [comment, marked as answer]
This is funkin master github 
If you have issues with polymod, open the issue [here](https://github.com/larsiusprime/polymod/issues).
